README.md

```diff
@@ -1705,6 +1705,7 @@ A [search engine](https://en.wikipedia.org/wiki/Search_engine_(computing)) is an
 - [Gigablast](https://www.gigablast.com/) - Open-source search engine. ([Source Code](https://github.com/gigablast/open-source-search-engine)) `Apache-2.0` `C++`
 - [Jina](https://github.com/jina-ai/jina/) - Cloud-native neural search framework for any kind of data. `Apache-2.0` `Python`
 - [librengine](https://github.com/liameno/librengine) - Private web search engine. `GPL-3.0` `C++`
+- [LibreX](https://github.com/hnhx/librex) -  Framework and javascript free privacy respecting meta search engine. `AGPL-3.0` `PHP/Docker`
```

Comment:
```suggestion
- [LibreX](https://github.com/hnhx/librex) - Framework and javascript-free, privacy-respecting meta search engine. `AGPL-3.0` `PHP/Docker`
```

Reply:
Ah, you must be correct then. My mistake.

Reply:
Thanks, I fix it.

Reply:
> SearX / SearXNG follows a similar principle i would assume.

I think you are right. Searx describes itself as a meta-search engine, which can query the following search engines: https://searx.github.io/searx/admin/engines.html. It does not provide its own crawler. I will add the `⚠` symbol to Searx. Thanks